source/Attenuator/Bargraph.h

```diff
@@ -18,6 +18,9 @@
  *
  */
 
+#ifndef ATTENBARGRAPH_H
```

Comment:
For the naming I would prefer just using the filename (eg. BARGRAPH_H) as it's unexpected that someone would be including a file from a different device. And while I get the purpose of this define statement for the include and I know it's technically good form to prevent duplicate inclusions, I'm curious as to what this solves or benefits it provides.

Reply:
It's good practice, and one of my IDEs gets incredibly irate if the header files don't have some form of duplication guard in place (it will error out on any attempt to use macros, for example `SYSTEM_MODE = MODE_SUPER_HERO;`.

Will switch these over to `#pragma once` in a bit, which should resolve the naming issue.